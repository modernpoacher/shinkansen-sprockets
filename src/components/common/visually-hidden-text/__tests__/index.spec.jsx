import React from 'react'
import renderer from 'react-test-renderer'

import Component from 'shinkansen-sprockets/components/common/visually-hidden-text'

describe('shinkansen-sprockets/components/common/visually-hidden-text', () => {
  describe('<Component />', () => {
    describe('With required props', () => {
      it('renders', () => {
        return expect(renderer.create(<Component visuallyHiddenText='MOCK VISUALLY HIDDEN TEXT' />).toJSON())
          .toMatchSnapshot()
      })
    })
  })
})
