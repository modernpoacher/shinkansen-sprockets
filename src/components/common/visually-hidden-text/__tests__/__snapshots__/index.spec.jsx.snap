// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shinkansen-sprockets/components/common/visually-hidden-text <Component /> With required props renders 1`] = `
<span
  className="visually-hidden-text"
>
  MOCK VISUALLY HIDDEN TEXT
</span>
`;
